Remotion/ObjectBinding/Web/UI/Controls/BocTreeNode.cs

```diff
@@ -220,14 +233,16 @@ public class BusinessObjectPropertyTreeNode: BocTreeNode
   /// <summary>
   ///   The <see cref="IBusinessObjectReferenceProperty"/> of this <see cref="BusinessObjectPropertyTreeNode"/>.
   /// </summary>
+  [AllowNull]
```

Comment:
why?

Reply:
Null can be set but not retrieved. Am I wrong?

Reply:
Smae discussion: https://github.com/re-motion/Framework/pull/235#discussion_r753320779

Reply:
But in this case, `EnsureProperty` really ensures that `_property` is not `null`.

Reply:
Updated: given EnsureProperty cannot ensure that _property != null (at this time), the Property getter would be nullable anyway.

Reply:
There are 2 `EnsureProperty` methods in this file. This one ensures it - if I don't see the case where it doesn't, please enlighten me.

Reply:
discussed.